playbooks/openshift-hosted/private/openshift_hosted_registry.yml

```diff
@@ -2,9 +2,13 @@
 - name: Create Hosted Resources - registry
   hosts: oo_first_master
   tasks:
-  - set_fact:
+  - name: Set openshift_hosted_registry_registryurl to first master registry_url
```

Comment:
We can get rid of both of these set_facts.

both registry and router url are never undefined.  They are defined in defaults/main.yml

Looks like openshift.master.registry_url is based on "oreg_url_master | default(oreg_url)"

We should work this logic into the defaults and remove the reference to the openshift.master.registry_url here.